src/components/Products.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Product from "./Product";
+import productData from "../data/productData";
+
+class Products extends React.Component {
+
+    handleSubmit=(e)=>{
+        e.preventDefault();
+
+    }
+  
+
+    render() { 
+        return (
+            <React.Fragment >
+                <form onSubmit={this.handleSubmit}>
+                  <section className="products">
+                     {productData.map(product=>  <Product product={product} addCart={this.props.AddCart} key={product.id}/>)}
+                  </section>
+                </form>
+            </React.Fragment>
+        );
```

Comment:
```suggestion
        return (
                  <section className="products">
                     {productData.map(product=>  <Product product={product} addCart={this.props.AddCart} key={product.id}/>)}
                  </section>
        );
```
This "products" section doesn't need to be wrapped in a form or a `React.Fragment`. There are no inputs for the user to interact with, so it doesn't make sense to have a form here. The fragment is unnecessary too because you're only returning one element anyway.